src/components/blog/blog-page.js

```diff
@@ -15,26 +14,37 @@ class Blog extends Component {
 
   render() {
     if (this.props.apiBlogData.loaded) {
-      let MEDIUM_PUBLICATION = this.props.apiInfoData.footerData.mediumSlug
-      return (
-        <Container styleName="common.margin">
-          <Card.Group itemsPerRow={3} stackable doubling>
-            {this.props.apiBlogData.data.map((info, id) => (
-              <BlogDetail info={info} key={id} />
-            ))}
-          </Card.Group>
-          <Segment basic padded textAlign="center">
-            <Icon
-              name="medium"
-              size="large"
-              link={true}
-              onClick={() =>
-                window.open(`${MEDIUM_URL}${MEDIUM_PUBLICATION}`, '_blank')
-              }
-            />
-          </Segment>
-        </Container>
-      )
+      if (this.props.apiBlogData.data && this.props.apiBlogData.data.length) {
+        let MEDIUM_PUBLICATION = this.props.apiInfoData.footerData.mediumSlug
+
+        return (
+          <Container styleName="common.margin">
+            <Card.Group itemsPerRow={3} stackable doubling>
+              {this.props.apiBlogData.data.map((info, id) => (
+                <BlogDetail info={info} key={id} />
+              ))}
+            </Card.Group>
+            <Segment basic padded textAlign="center">
+              <Icon
+                name="medium"
+                size="large"
+                link={true}
+                onClick={() =>
+                  window.open(`${MEDIUM_URL}${MEDIUM_PUBLICATION}`, "_blank")
```

Comment:
`'_blank'`

Reply:
but it is in jsx element so shouldn't it be double quotes?

Reply:
As soon as you enter `{ ... }`, you start writing JS.

Reply:
Ok.
Added changes